include/vrv/iomusxml.h

```diff
@@ -94,7 +94,7 @@ namespace musicxml {
 
     class EndingInfo {
     public:
-        EndingInfo(std::string endingNumber, std::string endingType, std::string endingText)
+        EndingInfo(std::string const &endingNumber, std::string const &endingType, std::string const &endingText)
```

Comment:
Sorry to be picky, could you change to `const std::string &endingNumber`? No difference, but I like these things to be consistent. Thanks

Reply:
Alright. I will do.

Reply:
I have another question, that I came across: there are two versions of this enumartion in the verovio project:  `LINESTARTENDSYMBOL_none` and `LINESTARTENDSYMBOL_NONE`. Should all be upper case?

<img width="477" alt="grafik" src="https://user-images.githubusercontent.com/45632600/57449421-2122d780-725c-11e9-8f40-49617bd96093.png">

Reply:
The one with the capital letters means an undefined end symbol, and the lower case one means that there is no end symbol.

In libmei/atttypes.h:

```
/**
 * MEI data.LINESTARTENDSYMBOL
 */
enum data_LINESTARTENDSYMBOL {
    LINESTARTENDSYMBOL_NONE = 0,
    LINESTARTENDSYMBOL_angledown,
    LINESTARTENDSYMBOL_angleup,
    LINESTARTENDSYMBOL_angleright,
    LINESTARTENDSYMBOL_angleleft,
    LINESTARTENDSYMBOL_arrow,
    LINESTARTENDSYMBOL_arrowopen,
    LINESTARTENDSYMBOL_arrowwhite,
    LINESTARTENDSYMBOL_harpoonleft,
    LINESTARTENDSYMBOL_harpoonright,
    LINESTARTENDSYMBOL_H,
    LINESTARTENDSYMBOL_N,
    LINESTARTENDSYMBOL_Th,
    LINESTARTENDSYMBOL_ThRetro,
    LINESTARTENDSYMBOL_ThRetroInv,
    LINESTARTENDSYMBOL_ThInv,
    LINESTARTENDSYMBOL_T,
    LINESTARTENDSYMBOL_TInv,
    LINESTARTENDSYMBOL_CH,
    LINESTARTENDSYMBOL_RH,
    LINESTARTENDSYMBOL_none,
    LINESTARTENDSYMBOL_MAX
};
```

Reply:
Ah, ok. I did not know the difference before. Thanks!

Reply:
Yes. _NONE mean there will be no attribute in the MEI output. With _none there will be an attribute with `"none"`.